frontend/src/components/donation_page/index.jsx

```diff
@@ -34,7 +34,7 @@ class Donate extends Component {
           </p>
 
           <div className="mx-auto donate">
-            <form action="https://www.paypal.com/" method="post" target="_top">
+            <form action="https://www.paypal.com/" method="post" target="_blank">
```

Comment:
Will target="_blank" open security vulnerabilities like it will for <a> tags? As far as I can tell the rel attribute used to fix that issue in <a> tags does not exist for <form> tags.